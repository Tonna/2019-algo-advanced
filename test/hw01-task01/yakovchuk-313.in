25
1 1 1 1 1 1 1 5 5 1 2 7 6 7 9 12 4 11 7 2 16 1 20 16
4
17 7
8 6
7 1
10 8
