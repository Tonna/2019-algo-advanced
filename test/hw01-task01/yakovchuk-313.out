30
13
7
12
